plugins/remove-in-allns.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: remove-in-allns
+spec:
+  version: v0.0.2
+  homepage: https://github.com/rajatjindal/kubectl-remove-in-allns
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/rajatjindal/kubectl-remove-in-allns/releases/download/v0.0.2/darwin-amd64-v0.0.2.tar.gz
+    sha256: faf99884619da7d6455f500eb2993790e75c7af84aa6e08a138846db45a6fa23
+    files:
+    - from: "*"
+      to: "."
+    bin: kubectl-remove-in-allns
+  - selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/rajatjindal/kubectl-remove-in-allns/releases/download/v0.0.2/linux-amd64-v0.0.2.tar.gz
+    sha256: 630be54a643fbceb7acfb6caf8794679f1f2a04e3146d440bcde0c21ec37b1b7
+    files:
+    - from: "*"
+      to: "."
+    bin: kubectl-remove-in-allns
+  shortDescription: Remove the requested resource from all the namespaces.
+  caveats: |
+    This is potentially disruptive operation. Not recommended for use in Production and use at your own risk.
+
+    supports following resources right now (PR welcome for supporting other resource types):
```

Comment:
given this caveat I'm questioning the usefulness of this.

it's often an anti-pattern to remove something from multiple namespaces given the namespaces can be a prod/staging/dev boundary.

Reply:
I wrote this as an enduser requested on kubernetes repo. I can understand if we don't want to include this in krew plugin list.